pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -60,6 +60,7 @@ func (c *Cluster) getOSDLabels(osd OSDInfo, failureDomainValue string, portable
 	labels[portableKey] = strconv.FormatBool(portable)
 	labels[deviceClass] = osd.DeviceClass
 	labels[osdStore] = osd.Store
+	labels["device-type"] = osd.DeviceType
```

Comment:
```suggestion
	labels[deviceType] = osd.DeviceType
```

Reply:
`GetDiskDeviceType()` will always return a value and never be blank. In your test cluster then why was the type blank? If this is only run in the osd prepare job, during an upgrade I could see how it would be blank. How about if we just don't set the deviceType label if it's blank? Anyway, nothing relies on this label, right? It's just nice to have if it's not blank.

Reply:
@umangachapagain ask for the device-type request far back, https://github.com/rook/rook/issues/11059

Do you remember where is this info used?

Reply:
This was intended to be used for some alerts. I don't think we ended up using it though.